### DISCUSSION TITLE
[5장] 웹 서버 질문

### DISCUSSION THREAD

**Discussion by ahyeon-jung** [learning-with/book-reading-http-reading-perfect-guide#7] [General]
1. 웹 서버와 웹 애플리케이션 서버의 차이는 무엇인가요?
2. CDN(Content Delivery Network)란 무엇이며, 어떻게 작동하나요?
3. Apache, Nginx, IIS와 같은 주요 웹 서버의 차이는 무엇인가요? 각각의 장단점은 무엇인가요?
4. www.google.com을 주소창에 쳤을 때 화면이 나오기까지의 과정을 네트워크 관점으로 설명해주세요.

**Comment by EUNCHAEv1006** [comment]
흥미로운 질문 목록이네요

**Comment by EUNCHAEv1006** [comment]
4번 질문에 대해 더 공부하고 싶다면 이 [링크](https://d2.naver.com/helloworld/59361) 가 잘 정리가 되어 있으니 다들 보시면 좋겠어요
